drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -10,6 +10,8 @@
 #include <sys/util.h>
 #include <drivers/clock_control/stm32_clock_control.h>
 
+#include "stm32_hsem.h"
```

Comment:
This one is not technically needed since included in soc.h, which is included here.

Reply:
Done